napari/_app/actions/_layer_actions.py

```diff
@@ -0,0 +1,162 @@
+"""This module contains actions (functions) that operate on layers.
+
+Among other potential uses, these will populate the menu when you right-click
+on a layer in the LayerList.
+"""
+
+from __future__ import annotations
+
+from functools import partial
+from typing import TYPE_CHECKING, List
+
+from app_model.types import Action
+
+from ...layers import _layer_actions
+from ..constants import CommandId, MenuGroup, MenuId
+from ..context import LayerListContextKeys as LLCK
+
+if TYPE_CHECKING:
+    from app_model.types import MenuRuleDict
+
+
```

Comment:
Despite the type hinting I didn't immediately know what these dicts were defining... perhaps a small comment?

```suggestion
# The following dicts define groups to which menu items in the layer list context menu can belong
# see https://app-model.readthedocs.io/en/latest/types/#app_model.types.MenuRule for details
```

Reply:
like it! thank you